WordPress/gutenberg
# [Block Library - Query Loop]: Add multiple authors support

Resolves: https://github.com/WordPress/gutenberg/issues/26668

This PR adds support for filtering a `Query Loop` with multiple authors instead of just one, which is the current behavior.

Follow up of: https://github.com/WordPress/gutenberg/pull/37949

In addition `QueryControls` component usage is now removed completely.

## Testing instructions
1. Build again the plugin in order to update the files with the changes in `webpack` config.
1. Everything should work as before in a `Query Loop` whether we had set an `author` filter to the block or not.
2. Everything should work properly by adding multiple authors in the query filter.

## Notes

This PR includes one more change that affects the building step of the plugin.
We need to automatically rename some functions when they are called inside block files, but have been declared elsewhere. This way we can call Gutenberg override functions, but the block will still call the core function when updates are back ported. Specifically we do it now for `build_query_vars_from_query_block`, but this could be used with other similar functions with the same use case.
